src/IsaacApiTypes.tsx

```diff
@@ -511,7 +511,7 @@ export type Stage = "year_7_and_8" | "year_9" | "gcse" | "a_level" | "further_a"
 
 export type ExamBoard = "aqa" | "cie" | "edexcel" | "eduqas" | "ocr" | "wjec" | "all";
 
-export type Difficulty = "practice_1" | "practice_2" | "practice_3" | "challenge_1" | "challenge_2" | "challenge_3";
```

Comment:
Are these difficulties specifically associated with physics for some reason or has cs just not made use of them so far?